frontend/src/Launcher/Launcher.js

```diff
@@ -1,14 +1,17 @@
-import React, {useState, useEffect} from "react";
+import React from "react";
 import { PageSection, PageSectionVariants, Grid, GridItem } from "@patternfly/react-core";
 import OdhAppCard from "./components/OdhAppCard";
 import { useFetch } from "./hooks";
 
-
 function Launcher() {
+
+  const PORT = process.env.PORT || 8080;
```

Comment:
Use a relative link.  This should just be `/api/components`.  For dev, you will need to use [proxying](https://create-react-app.dev/docs/proxying-api-requests-in-development/).  Before submitting this.  Turn off your local server and test the deployed image on OpenShift.

You're not going to have access to process.end.PORT/IP when hosting this.  If you did want to use absolute links (which you don't need), you would use `window.location.protocol/host/port`.

Reply:
Sure, will update it to relative link. Thanks for sharing the links - its helpful :)